### PR TITLE
Less crappy Chrome DevTools and Tools homepages

### PR DESCRIPTION
**Pull request by pbakaus**

### CODE REVIEW

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 1]
```diff
@@ -5,119 +5,275 @@ title: "Chrome DevTools"
...
-    <div class="mdl-cell mdl-cell--6-col">
-      <p>The Chrome DevTools are a set of web authoring and debugging tools built into Google Chrome. Use the DevTools to efficiently track down layout issues, set JavaScript breakpoints, and get insights for code optimization.</p>
-      <div class="note">
-        Note: Use <a href="https://tools.google.com/dlpage/chromesxs">Chrome Canary</a> for the latest DevTools.
-      </div>
+<div class="wf-devtools-wrapper">
+
+  <div class="wf-subheading wf-devtools-header">
+    <div class="page-content">
+        <h1><img src="images/chrome-devtools-logo.png" alt="">Chrome DevTools</h1>
```
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: Could you add some alt text

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 2]
```diff
@@ -5,119 +5,275 @@ title: "Chrome DevTools"
...
+    <div class="page-content mdl-grid">
+      <div class="mdl-cell mdl-cell--6-col">
+        <p>The Chrome DevTools are a set of web authoring and debugging tools built into Google Chrome. Use the DevTools to iterate, debug and profile your site.</p>
+        <div class="note">
+          <a href="https://tools.google.com/dlpage/chromesxs">Chrome Canary</a> always has the latest DevTools.
+        </div>
+      </div>
+      <div class="mdl-cell mdl-cell--6-col">
+        <ul>
+          <li>Select <strong>More Tools > Developer Tools</strong> from the Chrome Menu.</li>
```
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: Can you switch this to More Tools &gt; Developer Tools

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 3]
```diff
@@ -5,119 +5,275 @@ title: "Chrome DevTools"
...
+          <li>Right-click on a page element and select Inspect</li>
+          <li>Use <kbd class="kbd">Ctrl/Cmd</kbd>+<kbd class="kbd">Shift</kbd>+<kbd class="kbd">I</kbd> (<a href="https://developer.chrome.com/devtools/docs/shortcuts">more shortcuts</a>)</li>
```
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: Is the class needed if it's the same as the tag?

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 4]
```diff
@@ -5,119 +5,275 @@ title: "Chrome DevTools"
...
-* [Device Mode](/web/tools/chrome-devtools/iterate/device-mode)
-* [Remote Debugging](/web/tools/chrome-devtools/debug/remote-debugging)
+  <div class="wf-devtools-alternate">
+    <div class="page-content">
+      <h2 class="mdl-typography--font-light">Hot Off the Press</h2>
+
+        <div class="android-card-container mdl-grid">
+            <div class="mdl-cell mdl-cell--4-col mdl-card mdl-shadow--3dp">
+              <div class="mdl-card__media">
+                <img src="images/teaser1.png">
```
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: Alt tag please

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 5]
```diff
@@ -5,119 +5,275 @@ title: "Chrome DevTools"
...
+              </div>
+              <div class="mdl-card__title">
+                 <h4 class="mdl-card__title-text">Aggregated Details</h4>
+              </div>
+              <div class="mdl-card__supporting-text">
+                <span class="mdl-typography--font-light mdl-typography--subhead">Find out which third party scripts make your page slow.</span>
+              </div>
+              <div class="mdl-card__actions">
+                 <a class="mdl-button mdl-js-button mdl-typography--text-uppercase" href="">
+                   Find out now
```
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: This doesn't have a href - is that intentional?

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 6]
```diff
@@ -5,119 +5,275 @@ title: "Chrome DevTools"
...
+                 </a>
+              </div>
+            </div>
+
+            <div class="mdl-cell mdl-cell--4-col mdl-card mdl-shadow--3dp">
+              <div class="mdl-card__media">
+                <img src="images/teaser2.png">
```
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: Alt tag place.

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 7]
```diff
@@ -5,119 +5,275 @@ title: "Chrome DevTools"
...
+              </div>
+              <div class="mdl-card__title">
+                 <h4 class="mdl-card__title-text">Color Palettes</h4>
+              </div>
+              <div class="mdl-card__supporting-text">
+                <span class="mdl-typography--font-light mdl-typography--subhead">Automatic, Material and custom color palettes in the color picker.</span>
+              </div>
+              <div class="mdl-card__actions">
+                 <a class="android-link mdl-button mdl-js-button mdl-typography--text-uppercase" href="">
```
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: No href.

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 8]
```diff
@@ -5,119 +5,275 @@ title: "Chrome DevTools"
...
+                   Explore the colors
+                 </a>
+              </div>
+            </div>
+
+            <div class="mdl-cell mdl-cell--4-col mdl-card mdl-shadow--3dp">
+              <div class="mdl-card__media">
+                <img src="images/teaser3.png">
```
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: Alt attribute

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 9]
```diff
@@ -5,119 +5,275 @@ title: "Chrome DevTools"
...
+              </div>
+              <div class="mdl-card__title">
+                 <h4 class="mdl-card__title-text">New main menu</h4>
+              </div>
+              <div class="mdl-card__supporting-text">
+                <span class="mdl-typography--font-light mdl-typography--subhead">Streamlined, easier docking, settings and help access.</span>
+              </div>
+              <div class="mdl-card__actions">
+                 <a class="android-link mdl-button mdl-js-button mdl-typography--text-uppercase" href="">
+                   Learn more
```
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: No href.

File: src/static/styles/_global.scss [hunk 10]
```diff
@@ -8,6 +8,7 @@
 @import "../third_party/material-design-lite/src/textfield/textfield";
 @import "../third_party/material-design-lite/src/card/card";
 @import "../third_party/material-design-lite/src/menu/menu";
+@import "../third_party/material-design-lite/src/tabs/tabs";
```
Comment (gauntface) [inline src/static/styles/_global.scss]: this should be in the tools specific stylesheet.

File: src/static/styles/partials/components/_wf-tools-devtools.scss [hunk 11]
```diff
@@ -1,52 +1,134 @@
-.wf-tools-panel {
```
Comment (gauntface) [inline src/static/styles/partials/components/_wf-tools-devtools.scss]: The way the majority of styles are working is that a component should be a specific element and the filename should match up to the classname.

This looks like it is styles specific to the page. Please just make a top level stylesheet and add these there.

File: src/content/en/tools/chrome-devtools/index.markdown (hunk 2 above)
Comment (pbakaus) [inline src/content/en/tools/chrome-devtools/index.markdown]: I'm guessing you want to say & g t; but Github messed it up ;)

File: src/content/en/tools/chrome-devtools/index.markdown (hunk 3 above)
Comment (pbakaus) [inline src/content/en/tools/chrome-devtools/index.markdown]: Yes. It was historically so through all of our docs for some reason and got ported over this way. We can probably fix this in the future but this PR is not the appropriate place.

Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: Please address it in this PR.

File: src/content/en/tools/chrome-devtools/index.markdown (hunk 2 above)
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: Lol thats the one.

File: src/static/styles/partials/components/_wf-tools-devtools.scss (hunk 11 above)
Comment (pbakaus) [inline src/static/styles/partials/components/_wf-tools-devtools.scss]: OK I just used what was already there. I've moved it to the top level tools stylesheet instead. There's no other page that puts styles in those files but I agree it makes sense.

**Comment by pbakaus** [comment]
PTAL. All should be good.

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 12]
```diff
@@ -5,119 +5,272 @@ title: "Chrome DevTools"
...
+        <p>The Chrome DevTools are a set of web authoring and debugging tools built into Google Chrome. Use the DevTools to iterate, debug and profile your site.</p>
+        <div class="note">
+          <a href="https://tools.google.com/dlpage/chromesxs">Chrome Canary</a> always has the latest DevTools.
+        </div>
+      </div>
+      <div class="mdl-cell mdl-cell--6-col">
+        <ul>
+          <li>Select <strong>More Tools &gt; Developer Tools</strong> from the Chrome Menu.</li>
+          <li>Right-click on a page element and select Inspect</li>
+          <li>Use <kbd class="kbd">Ctrl/Cmd</kbd>+<kbd class="kbd">Shift</kbd>+<kbd class="kbd">I</kbd> (<a href="https://developer.chrome.com/devtools/docs/shortcuts">more shortcuts</a>)</li>
```
Comment (gauntface) [inline src/content/en/tools/chrome-devtools/index.markdown]: Can we drop the kbd class here please.

**Comment by gauntface** [comment]
Just the kbd thing then looks all good - thanks Paul.

**Comment by gauntface** [comment]
LGTM just waiting on travis to confirm all is well.

**Pull request merged**